libs/shared/src/consts/handlebar-helpers/getTemplateVariables.ts

```diff
@@ -10,27 +10,48 @@ export interface IMustacheVariable {
 }
 
 export function getTemplateVariables(bod: any[]): IMustacheVariable[] {
+  const pairVariables = bod
+    .filter((body) => body.type === 'HashPair')
+    .flatMap((body) => {
+      const varName = body.value?.original as string;
+
+      if (!shouldAddVariable(varName)) {
+        return [];
+      }
+
+      return {
+        type: TemplateVariableTypeEnum.STRING,
+        name: body.value?.original as string,
+        defaultValue: '',
+        required: false,
+      };
+    });
+
```

Comment:
This could be used in the future for other handlebars helpers that we might want to allow variables assignments in.

Reply:
maybe we should extract it to a reusable function then?